ua.org.brezblock.q4wine.appdata.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2022 Artem Polishchuk <ego.cordatus@gmail.com> -->
+<component type="desktop-application">
+  <id>ua.org.brezblock.q4wine</id>
+
+  <name>q4wine</name>
+  <summary>Utility for Wine applications and prefixes management</summary>
+  <summary xml:lang="cz">užitečný program pro programy wine a správu předpon</summary>
+  <summary xml:lang="de">Werkzeug zur Verwaltung der Wine-Anwendungen und -Präfixe</summary>
+  <summary xml:lang="en">Utility for Wine applications and prefixes management</summary>
+  <summary xml:lang="es">Utilidad para el manejo de aplicaciones y prefijos de wine</summary>
+  <summary xml:lang="it">Programma per gestire le applicazioni con Wine e i prefissi</summary>
+  <summary xml:lang="ru">Программа для настройки приложений и управления префиксами Wine</summary>
+  <summary xml:lang="uk">Програма для налаштування програм та керування префіксами Wine</summary>
+
+  <translation type="qt">q4wine/l10n/q4wine</translation>
```

Comment:
```suggestion
  <translation type="gettext">q4wine</translation>
```

Reply:
I am however not 100% sure about this change as mentioned here: https://github.com/flathub/ua.org.brezblock.q4wine/pull/1#issuecomment-1210665347

Reply:
This appdata tag has nothing to do with translations in app itself. It show in front-ends like Gnome Software on which languages app translated. This feature works in earlier version on Gnome Software but i don't know why it doesn't exist.

Reply:
Sorry, I was talking about translated strings in GNOME Software. But I will probably remove this change since I am not 100% sure about it.